### ISSUE TITLE
[Instructions for Commenting on WCAG 2 Documents] Says WCAG 2 is not developed anymore, despite WCAG 2.2 being developed right now

### ISSUE THREAD

**Issue by yatil** [w3c/wai-intro-wcag#181]
> Note that comments on WCAG 2 cannot impact the standard since it is no longer in development, although we will still take them into consideration as described below.

This is probably a little misleading.

**Comment by yatil** [comment]
I guess this comment has now been outpaced by reality.
